api/src/services/job_handlers/azure_container_instances/__init__.py

```diff
@@ -100,7 +107,7 @@ def start(self) -> str:
 
         # Configure the container group
         group = ContainerGroup(
-            location=self.job_entity.get("azure-location", "norwayeast"),
+            location=self.job_entity.get("azure-location", "northeurope"),
```

Comment:
got an error when using norwayeast and norwaywest - the compue resources was not available in those 2 regions